ynput/OpenPype
# Max: Implementation of the validator for model name

## Changelog Description
Implement the validator for model name in 3dsmax

## Additional info
The validation regex by default is `(.*)_(?P<subset>.*)_(GEO)` and 
your model name should be {Some name}_{SubsetName}_GEO
## Testing notes: 
1. Go to OP/AYON Settings
2. Turn on Validate Model Name
3. Set your preferred validation regex for checking your model name
4. Validate Model Name would check your name accordingly with your validation regex
